### PR TITLE
Fix copy article with live generation

### PR DESCRIPTION
**Pull request by alexander-schranz**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no 
| Fixed tickets | fixes # 
| Related issues/PRs | https://github.com/sulu/SuluArticleBundle/pull/610
| License | MIT
| Documentation PR | sulu/sulu-docs# 

#### What's in this PR?

Fix copy article with live generation.

#### Why?

When using live generating the route generating currently ends in some error. For copy we can just reuse the previous document routepath which will correclty postfix.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -144,7 +144,7 @@
     "conflict": {
         "doctrine/dbal": "2.7.0",
         "doctrine/lexer": "1.0.0",
-        "doctrine/orm": "2.10.0",
+        "doctrine/orm": "2.10.0 || 2.14.2",
```
Comment (alexander-schranz) [inline composer.json:147]: current doctrine/orm release has a bc break to gedmo extensions

**Comment by wachterjohannes** [comment]
@alexander-schranz LGTM but the tests fail quite hard :)

**Review by wachterjohannes — APPROVED** [review]

**Comment by alexander-schranz** [comment]
8.1 fails because of `@dev` dependency. `8.0` and `7.2` stable releases are now fixed. Will merge and tag a release soonish to have the conflict in it.

File: composer.json (hunk 1 above)
Comment (alexander-schranz) [reply composer.json:147]: fixes #7057

**Pull request merged**